Travel Scheduler/View Controllers/MapViewController.m

```diff
@@ -0,0 +1,239 @@
+//
+//  MapViewController.m
+//  Travel Scheduler
+//
+//  Created by frankboamps on 7/31/19.
+//  Copyright © 2019 aliu18. All rights reserved.
+//
+
+#import "MapViewController.h"
+#import "Place.h"
+#import <CoreLocation/CoreLocation.h>
+#import "TravelSchedulerHelper.h"
+
+@interface MapViewController () <CLLocationManagerDelegate>
+
+- (NSArray*) calculateRoutesFrom:(CLLocationCoordinate2D) from to: (CLLocationCoordinate2D) to;
+
+@end
+
+@implementation MapViewController
+
+#pragma mark - View Controller LifeCycle
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    self.title = @"Trip Overview";
+    [self.navigationController.navigationBar setTitleTextAttributes:
+     @{NSForegroundColorAttributeName:[UIColor darkGrayColor],
+       NSFontAttributeName:[UIFont fontWithName:@"Gotham-Light" size:21]}];
+    [self loadMapView];
+    self.arrayOfAnnotations = [[NSMutableArray alloc] init];
+    for (Place *place in self.placesFromSchedule){
+        [self addingPlaceMarkers:place withColor: [UIColor redColor]];
+        NSLog(@"%@", place.name);
+    }
+    self.view = _mainMapView;
+    [_mainMapView setDelegate:self];
+    self.buttonToNavigation = makeScheduleButton(@"Click For Trip Navigation");
+    self.buttonToNavigation.alpha = 1.0;
+    [self.buttonToNavigation addTarget:self action:@selector(navigation) forControlEvents:UIControlEventTouchUpInside];
+    [self.view addSubview:self.buttonToNavigation];
+    
+    int i;
+    for  (i = 0; i < [self.arrayOfAnnotations count] - 1; i++){
```

Comment:
I think the common practice is to do: for (int i = 0; i < ...;...) so you don't need line 44 if you instantiate the int within the for loop.